src/Controller/OcrController.php

```diff
@@ -220,16 +221,22 @@ public function homeAction(): Response
      */
     public function apiAction(): JsonResponse
     {
-        $this->setup();
-        $result = $this->getResult(EngineBase::WARN_ON_INVALID_LANGS);
-        $responseParams = array_merge(static::$params, [
-            'text' => $result->getText(),
-        ]);
-        $warnings = $result->getWarnings();
-        if ($warnings) {
-            $responseParams['warnings'] = $warnings;
+        try {
+            $this->setup();
```

Comment:
The exception is actually only coming from this method call isn't it? It might be better to just try/catch around this then, because the other calls below already have functioning error handling.